test/schemas/dependents-benefits/schema.spec.js

```diff
@@ -80,6 +79,18 @@ describe('dependents benefits schema', () => {
     }]
   });
 
+  schemaTestHelper.testValidAndInvalid('toursOfDuty', {
+    valid: [[{
```

Comment:
Took me entirely too long to figure out an array with an object _inside_ the `valid` array. 0.o